### PR TITLE
script for generating user test certs for PIVKey cards

### PR DESCRIPTION
**Pull request by dandds**
I made a little script that can generate the cert, key, and PFX package for user certs for the CAC.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -39,6 +39,9 @@ config/dev.ini
 /crl-tmp
 *.bk
 
+# test CA config
+ssl/client-certs/*.srl
```
Comment (patricksmithdds) [inline .gitignore]: Should this be `*.crl`? Or, how does a .srl get generated?

File: script/make-test-cac [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# script/make-test-cac:  Set up a test CAC card.
+# Usage:
+# ./script/make-test-cac [DOD identifier string] [user email] [certificate name]
```
Comment (patricksmithdds) [inline script/make-test-cac]: I was a little thrown off about "certificate name" -- is that just the filename of the generated cert?

File: script/make-test-cac [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+# i.e.:
+# ./script/make-text-cac JONES.ANDY.1234567890 andy@example.com andy
```
Comment (patricksmithdds) [inline script/make-test-cac]: tiny typo: `make-text-cac` => `make-test-cac`

File: .gitignore (hunk 1 above)
Comment (dandds) [reply .gitignore]: `openssl` needs the CA to have some kind of serial number in order to sign client certs, and the `-CAcreateserial` option on the `x509` command autogenerates it for us. I added *.srl to the gitignore, too.

File: script/make-test-cac (hunk 2 above)
Comment (dandds) [reply script/make-test-cac]: Yeah, it uses that name for all the generated files: `andy.crt`, `andy.key`, `andy.pfx`. I'll update it to "output name".

File: .gitignore (hunk 1 above)
Comment (dandds) [reply .gitignore]: Oh that _is_ the gitignore, duh

**Pull request merged**